src/Language/PureScript/Errors.hs

```diff
@@ -687,6 +691,11 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
       paras [ line "An exhaustivity check was abandoned due to too many possible cases."
             , line "You may want to decomposing your data types into smaller types."
             ]
+    renderSimpleErrorMessage (UnusedImport name) =
+      line $ "Unused import of module " ++ runModuleName name
+
+    renderSimpleErrorMessage (UnusedExplicitImport name names) =
+      line $ "Unused identifiers in import of module " ++ runModuleName name ++ ": " ++ intercalate ", " names
```

Comment:
Also, I've been trying to reword some of the errors so they read more like full sentences rather than fragments lately. So maybe something like "The import of <module name> contains the following redundant references:"